modules/installation-minimum-resource-requirements.adoc

```diff
@@ -184,8 +184,7 @@ endif::ibm-z[]
 ifndef::openshift-origin[]
 |Compute
 ifdef::ibm-z,ibm-power,ibm-cloud-vpc[|{op-system}]
-ifndef::ibm-z,ibm-power,ibm-cloud-vpc,vsphere[|{op-system}, {op-system-base} 8.6, {op-system-base} 8.7, or {op-system-base} 8.8 ^[3]^]
-ifdef::vsphere[|{op-system}, {op-system-base} 8.6, {op-system-base} 8.7, or {op-system-base} 8.8 ^[2]^]
+ifndef::ibm-z,ibm-power,ibm-cloud-vpc[|{op-system}, {op-system-base} 8.6—{op-system-base} 8.9 ^[3]^]
```

Comment:
see previous comment, maybe another way to handle this would be to refer to it it as `{op-system-base} 8.x`? though i'm not sure what the guidance is for using x to denote the different point releases